packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -63,6 +86,44 @@ class ScrollBehavior {
     }
   }
 
+  /// Wraps the given widget based on the information provided by [ScrollableDetails].
+  ///
+  /// For example, based on the platform and provided details, this method
+  /// could wrap a given widget with a [GlowingOverscrollIndicator] to provide
+  /// visual feedback when the user overscrolls.
+  Widget buildViewportDecoration(
```

Comment:
@Hixie, you had asked about this signature in the design doc, I created `ScrollableDetails` as you suggested. I agree it will make this more future proof if we need more information later.